repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/logging/apache/format/stock/ResponseBytesClfHandler.java

```diff
@@ -19,19 +19,34 @@
  */
 package org.openrepose.commons.utils.logging.apache.format.stock;
 
+import org.openrepose.commons.utils.StringUtilities;
```

Comment:
why would you use our string utils instead of one of the standard ones? we are trying to get rid of ours.